Samples/PublicSamples/RecordingBot/docs/tutorials/deploy/1-aks.md

```diff
@@ -0,0 +1,776 @@
+sample recording bot. Before we can start to run commands in the Azure command line tool, we need
+to log in:
+
+```powershell
+az login --tenant 99999999-9999-9999-9999-999999999999
+```
+
+After running the command, we see the following message:
+
+```text
+The default web browser has been opened at https://login.microsoftonline.com/99999999-9999-9999-9999-999999999999/oauth2/v2.0/authorize. Please continue the login in the web browser. If no web browser is available or if the web browser fails to open, use device code flow with `az login --use-device-code`.
+```
+
+A browser window with the Microsoft login page should open, otherwise follow the advice of the output.
+
+![Login Page](../../images/screenshot-login.png)
+
+In the browser, we need to log in with our Microsoft Entra Id administrator account and accept the requested scopes.
+
+After successful log in, the output looks like this:
+
+```json
+[
+  {
+    "cloudName": "AzureCloud",
+    "homeTenantId": "99999999-9999-9999-9999-999999999999",
+    "id": "yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyyy",
+    "isDefault": true,
+    "managedByTenants": [],
+    "name": "recordingbotsubscription",
+    "state": "Enabled",
+    "tenantId": "99999999-9999-9999-9999-999999999999",
+    "user": {
+      "name": "user@xyz.com",
+      "type": "user"
+    }
+  }
+]
+```
+
+## Create Azure Resource Group
+
+Now we can start to create resources in our azure subscription.  
+Let us start with creating a resource group in our Azure Subscription.
+
+```powershell
+az group create 
+    --location westeurope 
+    --name recordingbottutorial 
+    --subscription "recordingbotsubscription"
+```
+
+The result in the command line should look something like:
+
+```json
+{
+  "id": "/subscriptions/yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyyy/resourceGroups/recordingbottutorial",
+  "location": "westeurope",
+  "managedBy": null,
+  "name": "recordingbottutorial",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+## Create Azure Kubernetes Cluster
+
+With our Resource Group set up, we create an AKS cluster. For the purpose of this tutorial we'll
+use the Free Tier. To further reduce costs, we also set the node count of the system nodepool to 1.
+Regarding the vm size of the system nodes in the system nodepool we choose the
+`standard_d2s_v3`-series, as this series is available at the most regions and some nodes in this
+series are available without requesting more quotas.
+
+> [!WARNING]  
+> Make sure to check the [pricing tiers](https://learn.microsoft.com/en-us/azure/aks/free-standard-pricing-tiers) and [pricing for VMs](https://azure.microsoft.com/en-us/pricing/details/virtual-machines/linux/) as costs may incur (even with the Free Tier).
+
+> [!WARNING]  
+> Check your quotas [per vm quotas](https://learn.microsoft.com/en-us/azure/quotas/per-vm-quota-requests) as in Azure quotas for vCPU cores might first be increased.

```

Comment:
If you use azure virtual machines you use vCores of a specific types the default quota of a subscription is 10 vCores of type d-series. If you want to use more v-Cores in azure you first have to request a higher quota so azure knows how v-cores their customers want to use in the next time and possibly update the hardware. However if you ran out of a v-core quota and try to create a VM it just doesn't work because you exceeded you're allowed quota of v-cores to create the VM

Reply:
I still think, just mentioning the quotas and providing a link is enough here.
Whatever the reader is trying to do with v-Cores is sort of out of scope.
If you really want to mention all this, it should be a whole sentence (or more) and not condenced in "as in Azure quotas for vCPU cores might first be increased"

Reply:
I think just mentioning it with a sentence should be enough because users who know about quotas already have experience with Azure and already did something with quotas, I'll try to simplify the sentence